### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Zelinskyi-Serhii**
- [DEMO LINK](https://Zelinskyi-Serhii.github.io/layout_catalog_hovers/)

### CODE REVIEW

**Review by kirill-vypirovskyi — APPROVED** [review]
Looks good for me

File: src/styles/blocks/catalog.scss [hunk 1]
```diff
@@ -11,15 +13,22 @@
...
     align-items: center;
     box-sizing: border-box;
+    cursor: pointer;
 
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: $transition-time;
```
Comment (illiarerun) [inline src/styles/blocks/catalog.scss:23]: you should ALWAYS specify every property in transition

File: src/styles/blocks/navigation.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    }
+  }
+
+  &__link {
+    color: inherit;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-size: 12px;
+    font-weight: 500;
+    transition: $transition-time;
```
Comment (illiarerun) [inline src/styles/blocks/navigation.scss:25]: same even here

**Review by illiarerun — CHANGES_REQUESTED** [review]

File: src/styles/blocks/catalog.scss [hunk 3]
```diff
@@ -11,15 +13,22 @@
...
     align-items: center;
     box-sizing: border-box;
+    cursor: pointer;
 
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all $transition-time;
```
Comment (illiarerun) [inline src/styles/blocks/catalog.scss:23]: don't use all in your transitions, better specify all properties to make them refactorable in future

**Review by illiarerun — CHANGES_REQUESTED** [review]

**Review by illiarerun — APPROVED** [review]

**Pull request closed without merging**